cppcache/test/util/chrono/durationTest.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <chrono>
+
+#include <gtest/gtest.h>
+
+#include <geode/util/chrono/duration.hpp>
+
+using namespace apache::geode::util::chrono::duration;
+
+TEST(util_chrono_durationTest, ceil) {
+  EXPECT_EQ(std::chrono::seconds(1),
+            _ceil<std::chrono::seconds>(std::chrono::milliseconds(1)));
+  EXPECT_EQ(std::chrono::milliseconds(1),
+            _ceil<std::chrono::milliseconds>(std::chrono::milliseconds(1)));
+  EXPECT_EQ(std::chrono::microseconds(1000),
+            _ceil<std::chrono::milliseconds>(std::chrono::milliseconds(1)));
+}
+
+TEST(util_chrono_durationTest, to_string) {
+  EXPECT_EQ("42h", to_string(std::chrono::hours(42)));
+  EXPECT_EQ("42min", to_string(std::chrono::minutes(42)));
+  EXPECT_EQ("42s", to_string(std::chrono::seconds(42)));
+  EXPECT_EQ("42ms", to_string(std::chrono::milliseconds(42)));
+  EXPECT_EQ("42us", to_string(std::chrono::microseconds(42)));
+  EXPECT_EQ("42ns", to_string(std::chrono::nanoseconds(42)));
+}
+
+TEST(util_chrono_durationTest, from_string) {
```

Comment:
Nice! Unittests!  What do negative numbers do?

Reply:
Good idea. Should give negative durations.